docs/getting-started/security-ui.asciidoc

```diff
@@ -65,42 +68,20 @@ image::images/getting-started-pg.png[Shows the Get started page]
-
-image::images/overview-pg.png[Shows the Overview page]
-
-From the live feed on the *Overview* page, you can quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.
-
-TIP: The *Security news* section provides you with the latest {elastic-sec} news to stay informed on new developments, learn about {elastic-sec} features, and more.
-
-image::images/live-feed-ov-page.png[Shows the Overview page]
-
-*Histograms*
+=== Overview dashboard
 
-Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on areas of interest in time-based histograms, select a region to reflect a date range, or select a preset value in the timepicker. In the **Stack by** dropdown, you can select specific parameters to visualize individual counts. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display the total counts by alert name within the specified time frame.
+The Overview dashboard provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation. Refer to <<overview-dashboard, Overview dashboard>> for more information.
 
-TIP: All Elastic Security histograms, graphs, and tables contain an **Inspect** button so you can examine the {es} queries used to retrieve data throughout
-the app.
+image::images/overview-pg.png[Overview dashboard]
 
-*Host and network events*
-
-View event and host counts specific to Elastic data shippers and apps, such as **Auditbeats** or **Elastic Endpoint Security**. Expand each category to view specific counts of hosts or network events related to the selected category.
-
-[role="screenshot"]
-image::images/events-count.png[Shows host and network events on the Overview page]
-
-*Threat Intelligence*
-
-The Threat Intelligence view on the Overview page provides a streamlined way to collect threat intelligence data for threat detection and matching.
-
-The view shows the total number of ingested threat indicators, enabled threat intelligence sources, and ingested threat indicators per source. To visualize the ingested threat indicator data, click the *Source* link for a threat intelligence source.
+[float]
+[[detection-response-dashboard-ui]]
+=== Detection & Response dashboard
 
-NOTE: For more information about connecting to threat intelligence sources, visit <<es-threat-intel-integrations, Enable threat intelligence integrations>>.
+The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
```

Comment:
```suggestion
The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
```
Just making the same suggestion I made earlier in the PR on another topic, to make it more consistent in case you decide to go with it.